README.md

```diff
@@ -66,6 +66,11 @@ Btw, nodist also works in your PowerShell, but you might first need to 'Unblock'
 ```
 
 ```
+> nodist 5
```

Comment:
This is fixed now. I'm not pulling this because I'm going to rewrite the examples anyway as part of the 0.8 release. Thank you anyhow!